pkg/api/steve/machine/ssh.go

```diff
@@ -40,7 +46,50 @@ func onError(rw http.ResponseWriter, _ *http.Request, code int, err error) {
 	rw.Write([]byte(err.Error()))
 }
 
+// ensureCAPIInitialized ensures CAPI factory is initialized, doing so only once
+func (s *sshClient) ensureCAPIInitialized(ctx context.Context) error {
```

Comment:
@tomleb Could we build something into wrangler that starts/stops handlers depending on the availability of CRDs?